recipes/grpc/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "1.38.0":
+    url: "https://github.com/grpc/grpc/archive/refs/tags/v1.38.0.tar.gz"
+    sha256: "ABD9E52C69000F2C051761CFA1F12D52D8B7647B6C66828A91D462E796F2AEDE"
```

Comment:
cosmetic: keep it lowercase to make recipe consistent

Reply:
ok